<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advancedSettingForm</class>
 <widget class="QWidget" name="advancedSettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>202</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Genome mutations chances</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Coefficients bonus</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>mutation:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinBoxMutationChance">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>energy for cell height:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QDoubleSpinBox" name="spinBoxBuffHeight">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>turning on mutated gene:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="spinBoxTrueMutationChance">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>maximum energy per cage height:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QDoubleSpinBox" name="spinBoxBuffHeightMax">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>length mutation:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="spinBoxLenMutationChance">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>age:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="spinBoxAge">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>energy limit:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QSpinBox" name="spinBoxEnergyLimit">
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="buttonMutationChanceApply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QPushButton" name="buttonMutationChanceApplyExit">
     <property name="text">
      <string>Apply and exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
